cltool/src/cltool_main.cpp

```diff
@@ -664,8 +700,9 @@ static int cltool_createHost()
     cout << "Shutting down..." << endl;
 
     // close the interface cleanly, this ensures serial port and any logging are shutdown properly
-    inertialSenseInterface.Close();
-    inertialSenseInterface.CloseServerConnection();
+    // Note the following two lines are called by the InertialSense destructor, which will be called when this function exits
+    // inertialSenseInterface.Close();
+    // inertialSenseInterface.CloseServerConnection();
```

Comment:
@kylemallory Should we just remove this commented out code?  We can still add a comment to the following:

We don't need to close the InertialSense class intererface here because it the InertialSense desctructor.